components/build/tekton-chains/security-context-fix.yaml

```diff
@@ -0,0 +1,27 @@
+#
+# The default install of chains includes the following
+# which (iiuc) works in kubernetes but not openshift.
+#
+#    # User 65532 is the distroless nonroot user ID
+#    runAsUser: 65532
+#    runAsGroup: 65532
+#
+# The suggested workaround is to run this:
+#   oc adm policy add-scc-to-user anyuid -z tekton-chains-controller
+#
+# but instead we'll try removing the runAsUser and runAsGroup
+# config.
+#
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: tekton-chains-controller
+  namespace: tekton-chains
+spec:
+  template:
+    spec:
+      containers:
+      - name: tekton-chains-controller
+        securityContext:
+          runAsGroup:
+          runAsUser:
```

Comment:
Didn't we require runAsUser: 0 because of a latest issue, or has that been fixed upstream?

Reply:
I'm not aware of that, but I'll do some googling.